EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+
+    ![S_BGM C_C106 segment](../../Assets/Screenshots/EDI/Operation/ExportDefBGM_C106.png "[S_BGM  C_C106 segment]")
+
+3. Check the specification for the fields that are required to be filled. In most cases, you must fill the document number in the *D_1004* row (To identify a document). This is a numerical value that must be unique. For this reason, it is to be determined dynamically by a data source mapping. 
+
+4. Click the ![Add string attribute](../../Assets/Icons/Plus03.png "[Add string attribute]") button to the right of the *D_1004* row.   
+     - To the right of the *D_1004* row, only the ![Delete node](../../Assets/Icons/Trash10.png "[Delete node]")(Delete node) button is available now, with which you can remove the data, if required. The field name and description are no longer displayed in italics.  
+     - To the right of the *S_BGM C_C106*, the ![Add condition](../../Assets/Icons/AddCondition.png "[Add condition]") (Add condition) button is displayed with which you can define a condition for the output of this segment in the data mapping later.
+    - The *S_BGM C_C106* segment is now highlighted with a small black border. This indicates that the segment contains data.   
+    - The string attribute has been marked for a later data mapping.
+    
+5. Click the [SAVE] button to save the message structure.   
+    The *Exporter definitions* view is displayed. All defined string attributes are added to the attribute mapping function in the *Data sources* tab.
+
+6. If desired, check if the string attribute has been really marked for a later data mapping. To do this, open the message definition again and click the *Data sources* tab.  
+
+7. Select the related data source.   
+    - The *Map attributes* wizard window is displayed.
+    - The EDI Export BGM/C106/1004 *Destination attribute* has been added to the attributes to be mapped later or directly afterwards, if desired. For detailed information, see [Map data source](./05_ManageDataSources.md#map-data-source). 
+
+       ![Map attributes](../../Assets/Screenshots/EDI/Operation/ExportDefBGM_C106Mapping.png "[Map Attributes]")
+
+8. Finalize the *Map attributes* wizard by clicking the [CONTINUE] buttons until you can click the [FINALIZE] button  
+   The *Exporter definition sources* view is displayed.
+
+9. Continue with [Specify message dates](#specify-message-dates) described below.
+
+
+
+## Specify message dates
+
+Specify the message dates your business partner requires.   
+
+Each invoice can have a lot of different dates, for example the invoice date, the expected delivery date, the tax point date, and much more. The Actindo EDIFACT message structure contains only one BTM segment in its raw state. However, for many business partners, multiple BTM segments must be completed. As soon as you have specified one value in a BTM segment, a further BTM segment is added automatically so that you can specify as much BTM segments as your business partner require.
```

Comment:
Vorschlag: "and many more" (many/a lot of different dates)

Reply:
done